### PR TITLE
Fix JC's comments and modify saving filename

### PR DESCRIPTION
**Pull request by GuillaumeTh**
Fix dark mode, weird status glitch and summary issue.

New feature: name for the output files (json, csv, pdf) named with the same name: date_username_folder-containing-html.(json/csv/pdf).

### CODE REVIEW

**Comment by GuillaumeTh** [comment]
@czotti If you want to check you can. Otherwise I will merge next week and create a new pre-release

File: dmriqcpy/template/libs/js/scripts.js [hunk 1]
```diff
@@ -754,7 +750,7 @@ function load_qc(){
                     test.push([curr["qc"], curr["filename"], curr["status"], curr["comments"]])
                     // update_summ(filename+'_comments_summ', document.getElementById(filename + "_comments").value);
```
Comment (czotti) [inline dmriqcpy/template/libs/js/scripts.js:751]: removes?

File: dmriqcpy/template/libs/js/scripts.js [hunk 2]
```diff
@@ -754,7 +750,7 @@ function load_qc(){
...
                     document.getElementById(filename + "_status").innerText = status;
-                    document.getElementById(filename + "_status").style.backgroundColor = color_dict[status];
+                    document.getElementById(filename + "_status").style.backgroundColor = color_dict[status.trim()];
                     // update_summ(filename+'_status_summ', document.getElementById(filename+"_status").outerHTML);
```
Comment (czotti) [inline dmriqcpy/template/libs/js/scripts.js:754]: removes?

File: dmriqcpy/template/libs/js/scripts.js [hunk 3]
```diff
@@ -801,8 +797,8 @@ function save_qc(){
     }
     if (person != null){
         var currentdate = new Date();
-        var datetime = currentdate.getDate() + "/"
-                        + (currentdate.getMonth()+1)  + "/"
+        var datetime = ("0" + currentdate.getDate()).slice(-2) + "/"
```
Comment (czotti) [inline dmriqcpy/template/libs/js/scripts.js:800]: You could use `String(currentdate.getDate()).padStart(2, "0")` choose the one you found cleaner.

File: dmriqcpy/template/libs/js/scripts.js [hunk 4]
```diff
@@ -835,11 +831,26 @@ function save_qc(){
...
 }
 
+function get_person()
+{
+    return person;
+}
+
+function get_filename(person){
+    var currentdate = new Date();
+    var date = currentdate.getFullYear().toString() + ("0" + (currentdate.getMonth() + 1)).slice(-2).toString() + ("0" + currentdate.getDate()).slice(-2).toString();
```
Comment (czotti) [inline dmriqcpy/template/libs/js/scripts.js:847]: Check previous comment for zero padding string.

**Review by czotti — COMMENTED** [review]
Some minor comments. Pretty nice thanks.

**Pull request merged**